docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -52,32 +53,45 @@ include::_attributes.adoc[] <3>
 +
 <1> Set the `id` value to be the same as the file name but without the extension. You can shorten this if the file name is too long.
 <2> For information about how to create a good title for each content type, see xref:{doc-guides}/doc-reference.adoc#titles-and-headings[Titles and headings] on the "Quarkus style and content guidelines" page.
-<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the table of contents is generated, and content renders in the website portal. 
+<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the table of contents is generated, and content renders in the website portal.
```

Comment:
Item <1> and item <4> are also needed to ensure that the table of contents is generated and the content renders in the website portal. Could we move this bit up to line 44 as per my suggestion comment above and here:

```suggestion
<3> The `_attributes.adoc` include is required to ensure that attributes get resolved.
```

Reply:
Updated. Thanks, @michelle-purcell. In the next preview build, would you please verify that I achieved the result you were seeking?